main.py

```diff
@@ -220,11 +220,15 @@ def vms(self):
         vms = []
 
         for line in out.splitlines():
-            m = re.search(r'"([a-zA-Z0-9-_]+)"', line)
-            if m:
-                name = m.group(1)
+            m = re.findall(r'([a-zA-Z0-9-_]+)', line)
+            if m[0] != "inaccessible":
+                name = m[0]  # m.group(1)
                 state = self.state(name)
                 vms.append((state, name))
+            else:
+                name = m[1]
```

Comment:
Your code may raise `IndexError` if you don't find any matches with the regex (is it possible?)

Reply:
The output of "vboxmanage list vms" (which is the command that use to list the vms) is always built out of two strings, so I'm not worried of that.
For example:
"<inaccessible>" {a7a3b199-9e96-432b-848e-d04c6944afb2}
"machine-beast-span-port-dev-beast-monitoring-span-port_default_1541504960831_93343" {08c8ed7c-17d9-4e87-aed7-026d217e05e4}

Reply:
In next versions we should add some documentation for the required output